config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
+  mount_devise_token_auth_for 'User', at: 'auth'
```

Comment:
wdyt about adding some api request tests? like with a valid user and password, an invalid user, invalid password, etc. and expect for the desired responses.
Btw I was looking at the project setup and looks like you did not created the project with `RSpec` as the testing tool, and by default it uses `minitest`
I don't know if there is another task for setting up the testing tool (or is out of the scope of the target project), but just to mention we commonly use rspec instead of minitest, and personally I would recommend you too 😅
https://github.com/rootstrap/tech-guides/tree/master/ruby/rspec